libs/langchain/langchain/document_loaders/text.py

```diff
@@ -37,15 +37,15 @@ def load(self) -> List[Document]:
         """Load from file path."""
         text = ""
         try:
-            with open(self.file_path, encoding=self.encoding) as f:
+            with open(self.file_path, encoding=self.encoding,errors='ignore') as f:
```

Comment:
`errors='ignore'` will cause a silent error while decoding for a lot of users and isn't the appropriate way of handling decoding issues.

Here is an example of text that can be decoded correctly, but due to usage of `errors='ignore'` the decoding will silently decode the text incorrectly.

```
text = """This is a text encoded using CP-1252. It contains special characters like ©, ®, and ™, which are part of the Windows-1252 character set."""

text.encode('cp1252').decode('utf-8', errors='ignore')
```